doc/doxygen/tutorial/tutorial.h.in

```diff
@@ -555,7 +555,7 @@
  *       <td>step-64</td>
  *       <td> Solving a Helmholtz problem using matrix-free methods on the GPU
  *       with MPI parallelization.
- *       <br/> Keywords: LinearAlgebra::CUDAWrappers, CUDAWrappers::MatrixFree
+ *       <br/> Keywords: LinearAlgebra::CUDAWrappers, Portable::MatrixFree, Portable::FEEvaluation, Kokkos
```

Comment:
We don't even mention Kokkos in step-64 but we should probably move the language from "Cuda" to "Kokkos" there as well.

Reply:
Correct, we are not mentioning it, but we should. We also currently rely on CUDAWrappers for the linear algebra.